src/Surfnet/StepupSelfService/SelfServiceBundle/Security/Authentication/Provider/SamlProvider.php

```diff
@@ -118,7 +119,7 @@ private function getSingleStringValue($attribute, AssertionAdapter $translatedAs
         $values = $translatedAssertion->getAttributeValue($attribute);
 
         if (empty($values)) {
-            throw new BadCredentialsException(sprintf('Missing value for required attribute "%s"', $attribute));
+            throw new MissingRequiredAttributeException(sprintf('Missing value for required attribute "%s"', $attribute));
```

Comment:
When manually triggering this exception, my Request ID is always empty in the error page. Maybe this is wanted behavior is this the case?

From a users perspective, it might make more sense to remove empty values from the details table on the error page.